src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/Et1VettingController.java

```diff
@@ -56,8 +61,48 @@ public ResponseEntity<CCDCallbackResponse> initialiseEt1Vetting(
         CaseDetails caseDetails = ccdRequest.getCaseDetails();
         et1VettingService.initialiseEt1Vetting(caseDetails);
 
-        return getCallbackRespEntityNoErrors(caseDetails.getCaseData());
+        CaseData caseData = caseDetails.getCaseData();
+        List<JurCodesTypeItem> jurCodesCollection = caseData.getJurCodesCollection();
+        if (jurCodesCollection != null) {
+            caseData.setExistingJurisdictionCodes(
+                et1VettingService.generateJurisdictionCodesHtml(jurCodesCollection));
+        }
+        et1VettingService.populateTribunalOfficeFields(caseData);
 
+        return getCallbackRespEntityNoErrors(caseData);
     }
 
-}
+    /**
+     * Mid callback event in the Jurisdiction code page
+     * Gets userToken as a parameter for security validation and ccdRequest data which has caseData as an object.
+     * Validates the jurisdiction code that caseworker has added against the existing codes
+     * to prevent duplicate entries.
+     * Populates the track allocation html, which depends on the existing jurisdiction codes.
```

Comment:
It's important that JavaDoc doesn't merely talk through what the code is doing, but rather what the method is intended to achieve and all the edge cases.  This is something I intend on doing a session on as most dev's (as did I for many years) struggle with this.
The first part is fine, but then you go in to the code detail.  You know the meaning more than me, but try something like this:

"Mid callback event in the Jurisdiction code page to set the track allocation upon the case as long based upon jurisdiction."

This may not be the best example, but hopefully it helps.  We can discuss if helpful ;)

Reply:
Updated, hopfully it's better now